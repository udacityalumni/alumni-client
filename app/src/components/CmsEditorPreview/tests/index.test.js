import CmsEditorPreview from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

describe('<CmsEditorPreview />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <CmsEditorPreview
        title="Foobar"
        content="Foobar@"
        isShowing
        onClose={e => e}
      />
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
