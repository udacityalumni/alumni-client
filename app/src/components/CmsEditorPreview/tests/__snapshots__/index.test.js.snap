exports[`<CmsEditorPreview /> should render with default props 1`] = `
<Layer
  align="center"
  closer={true}
  hidden={false}
  onClose={[Function onClose]}>
  <Box
    announce={false}
    direction="column"
    focusable={true}
    pad="large"
    responsive={true}
    size="large"
    tag="div">
    <Heading
      align="center"
      tag="h1">
      Foobar
    </Heading>
    <Markdown
      components={Object {}}
      content="Foobar@" />
  </Box>
</Layer>
`;
